lib/column/date.go

```diff
@@ -171,6 +218,19 @@ func (col *Date) AppendRow(v interface{}) error {
 	return nil
 }
 
+func (col *Date) parseDate(str string) (datetime time.Time, err error) {
+	defer func() {
+		if err == nil {
+			err = dateOverflow(minDate, maxDate, datetime, defaultDateFormat)
+		}
+	}()
+	datetime, err = time.Parse(defaultDateFormat, str)
+	if err == nil {
+		return datetime, nil
+	}
+	return time.Parse(defaultDateTimeFormat, str)
```

Comment:
any reason we need to call this twice? can't we just return the err and a time.Time{}?